package.json

```diff
@@ -45,6 +44,9 @@
     "emailjs-com": "^2.6.4",
     "firebase": "^8.1.1",
     "jquery": "^3.5.1",
+    "moment": "^2.29.1",
+    "ngx": "^2.4.0",
+    "ngx-moment": "^5.0.0",
```

Comment:
Check the function from angular framework first

Reply:
Please dont use another library : **ANAGULAR should provide that feature** to format a  time /date